libvcx/Cargo.toml

```diff
@@ -16,10 +16,8 @@ crate-type = ["staticlib","rlib", "cdylib"]
 [features]
 default = []
 pool_tests = []
-agency = [] # migrate these to v2 agency
 agency_pool_tests = []
-agency_v2 = []
-pool_legacy_agency_tests = [] # migrate the test using this
+agency_tests = []
 general_test = []
```

Comment:
Would it be difficult to rename this tag to `general_tests` for consistency with the plural?